vulnerabilities/importers/redhat.py

```diff
@@ -126,7 +141,7 @@ def to_advisory(advisory_data):
             rhsa_aggregate_severities = []
             if rhsa_data.get("cvrfdoc"):
                 # not all RHSA errata have a corresponding CVRF document
-                value = rhsa_data["cvrfdoc"]["aggregate_severity"]
+                value = get_item(rhsa_data, "cvrfdoc", "aggregate_severity")
```

Comment:
Let's handle the cases where there is no value
```suggestion
                value = get_item(rhsa_data, "cvrfdoc", "aggregate_severity")
                if value:
```